### PR TITLE
added respective files

### PR DESCRIPTION
**Pull request by yashashwini16**
PR #9 
PR #10 
I have added java algorithm in Algorithm/JAVA folder .
I have added python file to Leetcode/python folder

### CODE REVIEW

File: Leetcode/Python/find pivot_index.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Time Complexity : O(n)
```
Comment (codemii) [inline Leetcode/Python/find pivot_index.py:1]: Please don't use spaces in your filename. 
Rename it please.

**Review by codemii — CHANGES_REQUESTED** [review]

File: Algorithms/Java/bubble_sort.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+public class BubbleSortExample {
```
Comment (codemii) [inline Algorithms/Java/bubble_sort.java:1]: Please rename class name. Don't use example like keys.

**Review by codemii — CHANGES_REQUESTED** [review]

File: Leetcode/Python/find pivot_index.py (hunk 1 above)
Comment (yashashwini16) [reply Leetcode/Python/find pivot_index.py:1]: yep

File: Algorithms/Java/bubble_sort.java (hunk 2 above)
Comment (yashashwini16) [reply Algorithms/Java/bubble_sort.java:1]: sure I ll

**Pull request merged**